### PR TITLE
Redo API docs tags.

### PR DESCRIPTION
**Pull request by phiggins**
While reviewing #2054, I noticed the setup for tags and tag groups was more complicated than it needed to be and there was some unnecessary cruft present, mostly as a result of figuring out how tags worked across the proto/swagger/redoc interactions.

In this PR:

* Remove unnecessary tag setup that we're not using anymore.
* Move all the tag setup into one file.
* Use the default tags in the proto to remove an unnecessary level of redirection.

### CODE REVIEW

File: components/automate-gateway/api/compliance/reporting/reporting.proto [hunk 1]
```diff
@@ -67,9 +64,6 @@ service ReportingService {
 			resource: "compliance:reporting:report-ids"
 			action: "compliance:reportids:list"
 		};
-		option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
-			tags: "Reports";
-		};
```
Comment (phiggins) [inline components/automate-gateway/api/compliance/reporting/reporting.proto]: As a result of this unnecessary level of redirection, we forgot to update the tag of this endpoint (`ListReportIds`) so it wasn't being displayed when viewing the API docs in the browser.

**Comment by phiggins** [comment]
> undoing the tags stuff makes it so that all of the proto docs are visible. do we want to hide the ones we haven't yet actually documented?

It's my understanding that only things tagged with a tag in a tag group will show up in the UI, but I'll double check.

**Comment by phiggins** [comment]
I just rebuilt the docs and only see the Reporting and Stats endpoints in the UI, so I think this is doing what's intended. If you're seeing other behavior please let me know!

**Comment by vjeffrey** [comment]
oh interesting. i def saw them all this morning but i’ll pull things down and check again later!

**Pull request merged**